test/check-machines-consoles

```diff
@@ -67,6 +67,8 @@ class TestMachinesConsoles(VirtualMachinesCase):
         b.wait_in_text('.pf-c-expandable-section__content',
                        'Clicking "Launch remote viewer" will download')
 
+        b.wait_not_in_text('.pf-c-expandable-section__content', 'SSH tunnel needs to be set up on client')
```

Comment:
Needs a positive test case, too. You already asked about this, just opening a thread to track it.